x-pack/plugins/security_solution/public/flyout/entity_details/shared/components/insight_entity/misconfigurations_overview.tsx

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback } from 'react';
+import { css } from '@emotion/react';
+import { EuiFlexGroup, EuiFlexItem, EuiText, useEuiTheme } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { DistributionBar } from '@kbn/security-solution-distribution-bar';
+import { useMisconfigurationPreview } from '@kbn/cloud-security-posture';
+import { euiThemeVars } from '@kbn/ui-theme';
+import { i18n } from '@kbn/i18n';
+import { ExpandablePanel } from '../../../../shared/components/expandable_panel';
+
+export const MisconfigurationsOverview = ({ hostName }: { hostName: string }) => {
+  const queryHostName = useCallback(() => {
+    return {
+      bool: {
+        must: [],
+        filter: [
+          {
+            bool: {
+              should: [{ term: { 'host.name': { value: `${hostName}` } } }],
+              minimum_should_match: 1,
+            },
+          },
+        ],
+        should: [],
+        must_not: [],
+      },
+    };
+  }, [hostName]);
+
+  const { data } = useMisconfigurationPreview({
+    query: queryHostName(),
+    sort: [],
+    enabled: true,
+    pageSize: 1,
+  });
+
+  const passedFindings = data?.count.passed || 0;
+  const failedFindings = data?.count.failed || 0;
+  const { euiTheme } = useEuiTheme();
+  const getFindingsStats = (passedFindingsStats: number, failedFindingsStats: number) => {
+    if (passedFindingsStats === 0 && failedFindingsStats === 0) return [];
+    return [
+      {
+        key: i18n.translate(
+          'xpack.securitySolution.flyout.right.insights.misconfigurations.passedFindingsText',
+          {
+            defaultMessage: 'Passed findings',
+          }
+        ),
+        count: passedFindingsStats,
+        color: euiThemeVars.euiColorSuccess,
+      },
+      {
+        key: i18n.translate(
+          'xpack.securitySolution.flyout.right.insights.misconfigurations.failedFindingsText',
+          {
+            defaultMessage: 'Failed findings',
+          }
+        ),
+        count: failedFindingsStats,
+        color: euiThemeVars.euiColorVis9,
+      },
+    ];
+  };
+
+  const MisconfigurationEmptyState = () => {
+    return (
+      <EuiFlexItem>
+        <EuiFlexGroup direction="column" gutterSize="none">
+          <EuiFlexItem>
+            <EuiText
```

Comment:
Setting font size using CSS isn't a good practice. EuiText has a property for that size.

It also happens in several other places in this file.